.github/SECURITY.md

```diff
@@ -0,0 +1,22 @@
+# Security Policy
+
+The PrestaShop team and community take security bugs in PrestaShop seriously. We appreciate your efforts to responsibly disclose your findings
+
+## Supported Versions
+
+Security updates will typically only be applied to the latest release.
+
+## Reporting a Vulnerability
+
+Security issues can be reported by sending an email to security@prestashop.com, which will go to security team members. 
+The team will send a response indicating the next steps in handling your report. 
+After the initial reply to your report, the security team will keep you informed of the progress towards a fix and full announcement, and may ask for additional information or guidance.
+
+## Disclosure Policy
```

Comment:
This section talks about the process, not the disclosure 🤔?

Reply:
oups, yes, my bad, I change  that :)